lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -530,70 +541,57 @@ def rescore_problem_module_state(xmodule_instance_args, module_descriptor, stude
         event_transaction_id = create_new_event_transaction_id()
         set_event_transaction_type(GRADES_RESCORE_EVENT_TYPE)
 
-        result = instance.rescore_problem(only_if_higher=task_input['only_if_higher'])
+        result = rescore_method(only_if_higher=task_input['only_if_higher'])
```

Comment:
Won't you need to add a try-catch block surrounding this call, per our discussion?  Or, will that happen with Sandy's work?